lib/matplotlib/axes/_base.py

```diff
@@ -2498,6 +2503,52 @@ def handle_single_axis(scale, autoscaleon, shared_axes, interval,
     def _get_axis_list(self):
         return (self.xaxis, self.yaxis)
 
+    def _update_title_position(self, renderer):
+        """
+        Update the title position based on the bounding box enclosing
+        all the ticklabels and x-axis spine and xlabel...
+        """
+        _log.debug('update_title_pos')
+
+        if self._autotitlepos is not None and not self._autotitlepos:
+            _log.info('title position was updated manually, not adjusting')
```

Comment:
Maybe this and the following log should be `debug`.  Do we have a clear policy on this?

Reply:
I don't know what the policy is.  However if I stuck too many things on DEBUG, folks complain, so maybe DEBUG is really INFO in most people's minds.

Reply:
INFO is between DEBUG and WARNING, so I think of it as a way to provide a little extra information that a person might want, but in the matplotlib context, not something that should be provided with each `draw`.  It looks like it is presently being used for situations close to, but just below, the WARNING threshold--situations that are being handled adequately and probably don't require the user's attention.

Reply:
Makes perfect sense that we don't want INFO spammed each draw....